cli/src/semgrep/meta.py

```diff
@@ -99,7 +99,24 @@ def repo_name(self) -> str:
 
     @property
     def repo_url(self) -> Optional[str]:
-        return get_url_from_sstp_url(os.getenv("SEMGREP_REPO_URL"))
+        env = get_state().env
+        repo_url = os.getenv("SEMGREP_REPO_URL")
+        if not repo_url:
+            # if the repo URL was not explicitly provided, try getting it from git
+            # nosem: use-git-check-output-helper
+            rev_parse = subprocess.run(
+                ["git", "config", "--get", "remote.origin.url"],
```

Comment:
```suggestion
                ["git", "remote", "get-url", "origin"],
```

I don't actually know, but my hunch is this is the preferred way to get the remote URL

```
$ git config --get remote.origin.url
git@github.com:returntocorp/semgrep-app.git

$ git remote get-url origin
git@github.com:returntocorp/semgrep-app.git
```